WalletWasabi.Fluent/Helpers/LabelHelpers.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using WalletWasabi.Fluent.ViewModels.Wallets.Labels;
+using WalletWasabi.Wallets;
+
+namespace WalletWasabi.Fluent.Helpers;
+
+public static class LabelHelpers
```

Comment:
When I see the "Helper", "Service", "Manager" suffixes, I tend to think there's a problem with either naming or with design. It's not so important right now, but maybe we can come up with a better name.